src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.Net.verified.txt

```diff
@@ -11,6 +11,7 @@
 [assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("Akka.Cluster.Tools")]
 [assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("Akka.DependencyInjection")]
 [assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("Akka.DistributedData")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("Akka.Docs.Tests")]
```

Comment:
Had to make `InteralsVisibleTo` the `Akka.Docs.Tests` project in order to take advantage of Polyfill.